pkg/sync/v5/data/ConfigMap/openshift-azure-logging/fluentd-config.yaml

```diff
@@ -9,19 +9,58 @@ data:
       </storage>
       tag logs
     </source>
-    <match logs>
+     # drop unused columns
+    <filter logs>
+      @type record_transformer
+      remove_keys _BOOT_ID,_CAP_EFFECTIVE,_MACHINE_ID,_SOURCE_REALTIME_TIMESTAMP,_STREAM_ID,_TRANSPORT,_SOURCE_MONOTONIC_TIMESTAMP
+    </filter>
+    # split logs to generic, audit and kubernetes runtime logs
+    <match logs >
       @type rewrite_tag_filter
       <rule>
         key MESSAGE
         pattern audit\.k8s\.io
         tag audit
       </rule>
+      <rule>
+        key CONTAINER_NAME
+        pattern ^k8s_
+        tag kubernetes
+      </rule>
```

Comment:
We need this, otherwise logic becomes a little bit more complex. This keeps it simple.

Reply:
+ kubernetes_metadata is RH filter :)